app/code/community/Reverb/ReverbSync/Helper/Sync/Product.php

```diff
@@ -4,10 +4,17 @@ class Reverb_ReverbSync_Helper_Sync_Product extends Mage_Core_Helper_Data
 {
     const UNCAUGHT_EXCEPTION_INDIVIDUAL_PRODUCT_SYNC = 'Error attempting to sync product with id %s with Reverb: %s';
     const PRODUCT_EXCLUDED_FROM_SYNC = 'The "Sync to Reverb" value for this product has been set to "No"; this product can not be synced to Reverb as a result';
+    const ERROR_INVALID_PRODUCT_TYPE = "Only %s products can be synced.";
+    const ERROR_INVALID_PRODUCT = 'An attempt was made to sync an unloaded product to Reverb';
```

Comment:
@skwp  If someone called method executeSimpleProductSync() with some custom code and passed in a product which was not loaded (perhaps calling Mage::getModel('catalog/product')->load(12) when there is no product with id 12 in the database) or passed it a value which was not an object, this error will be thrown. This is an overly cautious error message, I don't expect this to be thrown